test/Common/StandardConnectionInfoProviderTests.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using System.Collections.Generic;
+using Microsoft.Azure.WebJobs.Extensions.DurableTask.Options;
+using Microsoft.Extensions.Configuration;
+using Xunit;
+
+namespace Microsoft.Azure.WebJobs.Extensions.DurableTask.Tests
+{
+    public class StandardConnectionInfoProviderTests
+    {
+        [Fact]
+        [Trait("Category", PlatformSpecificHelpers.TestCategory)]
+        public void Resolve()
+        {
+            IConfiguration config = new ConfigurationBuilder()
+                .AddInMemoryCollection(
+                    new KeyValuePair<string, string>[]
+                    {
+                        new KeyValuePair<string, string>($"MyConnectionString", "Foo=Bar;Baz"),
+                        new KeyValuePair<string, string>($"MyConnection:{nameof(AzureStorageAccountOptions.AccountName)}", "MyAccount"),
```

Comment:
I would prefer hard coding "accountName" instead of using nameof(). If the actual object was changed in the future, that could lead to a regression.

Reply:
Done